### DISCUSSION TITLE
Docker启动命令参数怎么填

### DISCUSSION THREAD

**Discussion by mxthd** [natfrp/wiki#306] [使用问题] [answered]
docker run \
-d \ # 后台运行
--restart=always \ # 自动启动
--pull=always \ # 自动更新
--name=sakura1 \ # 为容器设定一个名字
natfrp/frpc \
-f <启动参数> --remote_control <远程控制密码>

在官网上看启动命令，结合用户手册的启动参数，一直无法启动，请问有完整的填写了token和通道id的启动命令示例吗，让新手一键就能启动

**Comment by mxthd** [comment, marked as answer]
已解决

**Comment by franklonghai** [comment]
你好，我也遇见同样的问题，请问怎么解决的，能详细说说吗

**Reply by franklonghai** [reply]
你好，我是有正确创建的，以下是日志
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 61.159.93.110:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 61.159.93.110:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 61.159.93.110:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused
stdout: request failed: /client/get_config Post "https://natfrp-api.globalslb.net/client/get_config": dial tcp 175.6.13.45:443: connect: connection refused

**Reply by fakeboboliu** [reply]
您好，请尝试直接访问 https://natfrp-api.globalslb.net/
如果发生报错，说明您的网络提供商或网管部门直接屏蔽了我们的域名。

**Reply by franklonghai** [reply]
可以正常访问打开的

[a comment by an automated account is not shown]
